Modernize authn tasks for 1.5 API and auto-mTLS by default (#6546)
istio/istio.io: content/en/faq/security/accessing-non-istio-services.md

weight: 40
---

> Istio will try to detect whether the destination workload have Envoy proxy or not, and drop mutual TLS if it doesn't. You can also set the destination rule explicitly to disable mutual TLS. For example:

Reviewer: ```suggestion
Istio detects if the destination workload has an Envoy proxy and drops mutual TLS if it doesn't. Set an explicit destination rule to disable mutual TLS. For example:
```

> Istio will try to detect whether the destination workload have Envoy proxy or not, and drop mutual TLS if it doesn't. You can also set the destination rule explicitly to disable mutual TLS. For example:

Reviewer: ```suggestion
Istio detects if the destination workload has an Envoy proxy and drops mutual TLS if it doesn't. Set an explicit destination rule to disable mutual TLS. For example:
```
